### DISCUSSION TITLE
Welcome to libiio Discussions!

### DISCUSSION THREAD

**Discussion by pcercuei** [analogdevicesinc/libiio#873] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kchuck** [comment]
Hi,
I am using the AD9361 on both Xilinx ZED and Microchip Polarfire development boards.  I have plenty of information on the ZED board and use IIOLIB, but nothing on the Polarfire.  I would like to run IIOLIB on an ARM processor in the Polarfire chip.  Has anyone done this?

**Reply by tfcollins** [reply]
Hi @kchuck

We are aware of folks running things on PolarFire. However, I don't believe such projects are publicly available.

-Travis
